.env-dist

```diff
@@ -6,6 +6,6 @@ PORT=8000
 DJANGO_DEBUG=True
 DJANGO_ALLOWED_HOSTS=localhost,127.0.0.1,
 DJANGO_SITE_URL=http://localhost:8000
-AWS_ACCESS_KEY_ID=AKIAJYW43RKL7AFG6JOA
-AWS_SECRET_ACCESS_KEY=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
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
```

Comment:
Ah ok, nvm. :) We should squash these 2 commits so there's no trace of those keys.